apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/iverksetting/utbetaling/Utbetalingsvedtak.kt

```diff
@@ -17,7 +17,7 @@ data class Utbetalingsvedtak(
         fun fra(vedtak: Vedtak) =
             Utbetalingsvedtak(
                 vedtakId = vedtak.vedtakId,
-                sak = Sak(vedtak.sak.ident, vedtak.sak.id, Saktype.fraString(vedtak.sak.sakType)),
+                sak = Sak(vedtak.sak.ident, vedtak.sak.id, Saktype.fraString(vedtak.sak.sakType.toString())),
```

Comment:
Dette er en liten drawback av omskrivningen da utbetaling har sitt eget saksobjekt og Saktype som har validering i selve saktype objektet. Det blir utenfor scope for denne oppgaven.